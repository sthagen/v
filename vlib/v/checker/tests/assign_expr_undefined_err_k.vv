fn main() {
	mut a := map{'one': a}
	println(a)
}
