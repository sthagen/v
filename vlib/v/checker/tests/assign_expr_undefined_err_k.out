vlib/v/checker/tests/assign_expr_undefined_err_k.vv:2:22: error: undefined variable: `a`
    1 | fn main() {
    2 |     mut a := map{'one': a}
      |                         ^
    3 |     println(a)
    4 | }
